docs/_static/reuse/schema-troubleshoot.rst

```diff
@@ -0,0 +1,4 @@
+Have you used the correct version of the F5 schema?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Additions to the `F5 schema`_ made with each version release support the new features in that specific version. For example, if you use v1.3.0 of the Controller with v0.1.2 of the schema, the Controller's core functionality would be fine. You wouldn't, however, be able to use the features from `k8s-bigip-ctlr v1.3.0`_.
```

Comment:
We should include this link for guidance: http://clouddocs.f5.com/containers/v2/releases_and_versioning.html#schema-table

Reply:
That's where the "F5 schema" link goes (line 4).